packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js

```diff
@@ -107,6 +107,7 @@ export default class AgoraIpV4OrderController {
       .all({
         user: this.User.getUser(),
         services: this.Ipv4AgoraOrder.getServices(),
+        vrackServices: this.Ipv4AgoraOrder.getVrackService(), // toDo
```

Comment:
```suggestion
        vracks: this.Ipv4AgoraOrder.getVrackService(), // toDo
```

Don't use vrackServices term as it correspond to another product

Reply:
done